test/load-with-esm.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert');
+
+// ESM support is flagged on v12.x.
+const majorVersion = +process.version.split('.')[0].slice(1);
+if (majorVersion < 12) {
+  process.exit();
```

Comment:
this will exit the process and other test case can't be executed.

Reply:
Fixed, check `Node.js` version to determine whether skip in `before` hook.